### ISSUE TITLE
If the command that launched the daemon proccess is start with --tor-mode strict, the daemon runs all commands with --tor-mode strict

### ISSUE THREAD

**Issue by zmanian** [keybase/client#1983]
Steps to reproduce

1. kill the daemon process

2. run "keybase --tor-mode strict status"

3. run "keybase --tor-mode leaky login" This will fail because the daemon process will apply tor-mode strict to all future commands.

**Comment by oconnor663** [comment]
Ouch, I hadn't thought about this before. We have a number of different flags that interact poorly with daemon mode. Just filed https://keybase.atlassian.net/browse/CORE-2536 internally.

@maxtaco is this something you already have ideas about?

**Comment by maxtaco** [comment]
@oconnor663 yeah that's known.  We have a ticket to plumb through these options better, we took some shortcuts here, we'll fix it. For now, these features are frozen at the time of process launch.  If you want the old one-off process model, feel free to use `--standalone` mode in the mean time. Thanks for the feedback!
